README.md

```diff
@@ -1,3 +1,57 @@
 # REDCap to Harvest
 
 Dockerfile and tools for deriving Harvest applications from REDCap projects.
+
+Installation and Setup
+----------------------
+
+This tool runs using Docker. Install it if you don't have it:
+
+[Install Docker](http://docs.docker.com/installation/) 
+
+If you are using Mac, use boot2docker to run Docker.
+
+
+After docker is installed, enter the directory that contains this repository and build the Docker container:
+
+```bash
+docker build .
+```
+
+When the build is complete, find the image docker built:
+```bash
+docker images
+```
+You should see a list of images like this:
```

Comment:
They won't see that because you tagged it `swanijam/djredcap-harvest` locally, but not in the directions above. To alleviate all of this, I created a Docker hub repository for this repo. The instructions can be:

```
docker run -p 8000:8000 -v /path/to/files:/files dbhi/redcap-harvest
```

We generally assume the user has limited knowledge of we are doing here. So a minimal, single command that *makes it work* is important. If they *do* know what they are doing, then they can customize it however they want.